tests/suites/sparsebase/experiment/experiment_tests.cc

```diff
@@ -22,15 +28,23 @@ auto load_data_f( vector<string> & file_names ) {
   return r;
 };
 
-void preprocess_f(unordered_map<string, Format*> & data, std::any & params) {
-
-  data.emplace("ordered", A_reordered);
+void preprocess_f(unordered_map<string, Format*> & data, std::any & fparams, std::any & params) {
+  data.emplace("new_format", nullptr);
 };
 
 std::any kernel_f(unordered_map<string, Format*> & data, std::any & fparams, std::any & pparams, std::any & kparams) {
+    return {};
 };
 
 
 TEST(experiment, Basics){
- cout << "tstss" << endl;
+  ConcreteExperiment exp;
+  exp.AddDataLoader(load_data_f, {});
+  exp.AddPreprocess("id", preprocess_f, {});
+  exp.Run(2, true);
+  auto aux = exp.GetAuxiliary();
+  for(const auto & [id, data]: aux){
+    cout << id << endl;
+
```

Comment:
The test needs some ASSERT or EXPECT statements :)

Reply:
tests are incomplete for now